storage/pebble/constants.go

```diff
@@ -0,0 +1,24 @@
+package pebble
+
+import "github.com/onflow/flow-go/storage/pebble/registers"
+
+const (
+	// lookup keys for register heights (f,l)
+
+	// placeHolderHeight is an element of the height lookup keys of length HeightSuffixLen
+	// 10 bits per key yields a filter with <1% false positive rate.
+	placeHolderHeight = uint64(0)
+
+	// MinLookupKeyLen is a structure with at least 2 fixed bytes for:
+	// 1. '/' byte separator for owner
+	// 2. '/' byte for key (owner and key values are blank so both have 0 bytes before each '/')
+	// 3. the 8 byte space for uint64 in big endian for the block height
+	MinLookupKeyLen = 2 + registers.HeightSuffixLen
+
+	// prefixes
+	// codeRegister (r)
+	codeRegister byte = 0x72
+	// codeFirstBlockHeight and  codeLatestBlockHeight keys for register heights (f,l)
+	codeFirstBlockHeight  byte = 0x66
+	codeLatestBlockHeight byte = 0x6C
```

Comment:
I worry that using non-structured codes will make it hard to manage these when the lists grows. What do you think about using [sequential values](https://github.com/onflow/flow-go/blob/master/storage/badger/operation/prefix.go) like we do for badger?

Reply:
the initial reasoning I had for using characters was:
- I didn't anticipate too many unique data types for this DB instance
- I wanted to keep the keys human-readable for debugging

I put some more consideration to this and I think sequential values is fine, we don't have that many keys to remember and the debugging process will be the same if we have to remember ints instead anyway